flags/tests/test_jinja2tags.py

```diff
@@ -0,0 +1,36 @@
+from django.template import engines
+from django.test import TestCase, override_settings
+
+
+templates_override = [
+    {
+        'NAME': 'jinja-engine',
+        'BACKEND': 'django.template.backends.jinja2.Jinja2',
+        'OPTIONS': {
+            'extensions': [
+                'flags.jinja2tags.flags',
+            ],
+        }
+    },
+]
+
+
+@override_settings(
+    TEMPLATES=templates_override,
+    FLAGS={'MY_FLAG': {'boolean': 'True'}}
+)
+class FlagsExtensionTests(TestCase):
+    def setUp(self):
+        self.jinja_engine = engines['jinja-engine']
+
+    def test_jinja2_flag_enabled_tag(self):
```

Comment:
Can we add a unit test here that tests the passing of a keyword parameter to this tag? Also to [these Django template tags](https://github.com/cfpb/django-flags/blob/master/flags/tests/test_templatetags_feature_flags.py#L6)? This is after all the main change in this PR.